Cargo.toml

```diff
@@ -46,6 +46,30 @@ x25519-dalek = "0.5"
 yubihsm = { version = "0.29", features = ["setup", "usb"], optional = true }
 zeroize = "1"
 
+[dependencies.signatory]
```

Comment:
could this be against: https://github.com/crypto-com/signatory ?

Reply:
I change someting in signatory when I use it in kms, and there is a new PR for crypto-com/signatory, I can't use it until you accept the signatory PR.

Reply:
oh, ok -- it's merged now